cmd/server/pactasrv/authz.go

```diff
@@ -0,0 +1,139 @@
+package pactasrv
+
+import (
+	"context"
+
+	"github.com/RMI/pacta/oapierr"
+	"github.com/RMI/pacta/pacta"
+	"go.uber.org/zap"
+)
+
+type authzStatus struct {
+	primaryTargetID      string
+	primaryTargetType    pacta.AuditLogTargetType
+	primaryTargetOwnerID pacta.OwnerID
+
+	secondaryTargetID      *string
+	secondaryTargetType    *pacta.AuditLogTargetType
+	secondaryTargetOwnerID *pacta.OwnerID
```

Comment:
optional: Instead of using pointers here, which has awkward semantics, I'd just use empty strings to denote "empty", it's the Go-y way to do these things. We never have a valid ID/TargetType that's a blank string.

Reply:
SGTM.